cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,133 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import checkout from '../../support/fragments/checkout/checkout';
+import searchPane from '../../support/fragments/circulation-log/searchPane';
+import circulationRules from '../../support/fragments/circulation/circulation-rules';
+import noticePolicy, { NOTICE_ACTIONS } from '../../support/fragments/circulation/notice-policy';
+import noticePolicyTemplate, { TEMPLATE_CATEGORIES } from '../../support/fragments/circulation/notice-policy-template';
+import patronGroups from '../../support/fragments/settings/users/patronGroups';
+import topMenu from '../../support/fragments/topMenu';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import checkInActions from '../../support/fragments/check-in-actions/checkInActions';
+
+// TODO email checking
+describe('Recieving notice: Checkout', () => {
```

Comment:
there are several steps started from "Pay attention " . Double check with PO that you can play such steps through api without checking of implementation in GUI.

Reply:
changes executing to GUI

Reply:
and I can't see implementation of "Active check mark button selected automatically", for example.